docs/source/AdministratorGuide/Systems/DataManagement/dfc.rst

```diff
@@ -0,0 +1,80 @@
+.. _dfc:
+
+------------------
+Dirac File Catalog
+------------------
+
+The DIRAC File Catalog (DFC) is a full replica and metadata catalog integrated to DIRAC. It has a very modulare structure, allowing for several backends. The interaction with the backend is handled by `Managers` in such a way that the interface exposed to the users remains always the same.
+
+There are two main sets of managers:
+* the historical ones, offering the full range of functionalities and used by most VO
+* and the LHCb ones, where a subsets of the functionalities related to user defined metadata are not tested, but optimized for scaling and consistency. Any VO could of course use it.
+
+
+Installation
+------------
+
+The installation and configuration procedure changes slightly between the historical managers and the LHCb ones.
+
+The list of components you need to have installed is:
+
+   * FileCatalogDB: if you want the standard managers, you should use `FileCatalogDB.sql`, but `FilecatalogWithFkAndPsDB.sql` if you want the LHCb ones
+   * FileCatalogHandler: just the interface to the DB
```

Comment:
All this is already in here http://dirac.readthedocs.io/en/latest/AdministratorGuide/scalingAndLimitations.html
I would rather take the following approach "unless otherwise specified, any service can be replicated as many time as you wish"